egs/wsj/s5/steps/nnet3/compute_and_adjust_priors.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python
+
+# Copyright 2016    Pegah Ghahremani.
+# Apache 2.0.
+
+""" This script used to compute average posterior and re-adjust prior
```

Comment:
@pegahgh, can you explain why you are adding a python version of this script?  Where possible we prefer shell scripts to python scripts, they are much clearer.  Why is this script needed?